src/join-2.sol

```diff
@@ -20,19 +20,7 @@
 
 pragma solidity >=0.5.12;
 
-import "dss/lib.sol";
-
-interface VatLike {
-    function slip(bytes32, address, int256) external;
-}
-
-interface GemLike {
-    function decimals() external view returns (uint256);
-    function transfer(address, uint256) external;
-    function transferFrom(address, address, uint256) external;
-    function balanceOf(address) external view returns (uint256);
-    function allowance(address, address) external view returns (uint256);
-}
+import "./deps.sol";
```

Comment:
If this deps thing was made due errors when importing in `ilk-deployer.sol` I'd suggest instead using the approach of the test:
```
import {GemJoin2} from "./join-2.sol";
import {GemJoin3} from "./join-3.sol";
import {GemJoin4} from "./join-4.sol";
import {GemJoin5} from "./join-5.sol";
import {GemJoin6} from "./join-6.sol";
import {GemJoin7} from "./join-7.sol";
import {GemJoin8} from "./join-8.sol";
```
Which will avoid the conflicts and the need of this new file.